odoo/modules/loading.py

```diff
@@ -246,6 +246,12 @@ def load_module_graph(cr, graph, status=None, perform_checks=True,
             # update made to the schema or data so the tests can run
             # (separately in their own transaction)
             cr.commit()
+            if model_names:
+                cr.execute("""select model from ir_model where id NOT IN (select distinct model_id from ir_model_access) AND model IN %s""", (tuple(model_names),))
+                for [model] in cr.fetchall():
+                    if model in registry and not registry[model]._abstract:
```

Comment:
by construction, all `model_names` are in the registry
```suggestion
                    if not registry[model]._abstract:
```